src/test/java/com/gd/clinic/containers/ContainersEnvironment.java

```diff
@@ -0,0 +1,13 @@
+package com.gd.clinic.containers;
+
+import org.testcontainers.containers.PostgreSQLContainer;
+import org.testcontainers.junit.jupiter.Container;
+import org.testcontainers.junit.jupiter.Testcontainers;
+
+@Testcontainers
+public class ContainersEnvironment {
+
+    @Container
+    public static PostgreSQLContainer postgreSQLContainer = PostgresTestContainer.getInstance();
```

Comment:
testcontainers is really nice
but try not have warnings, you can add generic `<?>` here so that it dissapear

Reply:
That's right, they really made it easier! Fixed the issue this commit, added wildcard.